README.md

```diff
@@ -759,3 +760,49 @@ personalisation.put("reference_number", "13566");
 ```
 
 </details>
+
+## Get all received text messages
+
+```java
+ReceivedTextMessageList response = client.getReceievedTextMessages(olderThanId);
+```
+
+#### Response
+
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+If the request is successful, `response` will be a `ReceivedTextMessageList`:
+
+```java
+private List<ReceivedTextMessage> receivedTextMessages;
+private String currentPageLink;
+private String nextPageLink;
+```
+
+A `ReceivedTextMessage` will have the following properties -
+
+```java
+private UUID id;
+private String notifyNumber;
+private String userNumber;
+private UUID serviceId;
+private String content;
+private DateTime createdAt;
+
+```
+</details>
+
+#### Arguments
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+	
+##### `olderThanId`
+
+A String parameter, if null or empty all messages are returned. Otherwise you can filter to fetch all received text messages older than the given id.
```

Comment:
Should probably indicate that only 250 messages will be returned and not all received text messages.